app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,19 @@
+class CommentsController < ApplicationController
+  def create
+    @request = Request.find(params[:request_id])
+    @comment = Comment.new(comment_params)
+    @comment.request = @request
+    @comment.author = current_user
+    if @comment.save
+      redirect_to request_path(@request)
+    else
+      redirect_back(fallback_location: root_path)
```

Comment:
if it doesn't save, you want to render the page where the form was submitted.
with this: 
```
flash[:alert] = "Information missing."
```